lib/images.sh

```diff
@@ -11,8 +11,8 @@ elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/}
   export IMAGE_USERNAME=${IMAGE_USERNAME:-core}
 elif [[ "${IMAGE_OS}" == "Centos" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-CentOS-7-x86_64-GenericCloud-1907.qcow2}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-http://cloud.centos.org/centos/7/images}
+  export IMAGE_NAME=${IMAGE_NAME:-centos-updated.qcow2}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-http://artifactory.nordix.org/artifactory/airship/images/centos.qcow2}
```

Comment:
This needs to be changed to CentOS8

Reply:
yes, I'll for now I just want to make sure it is going to work with CentOS8